eng/pipelines/coreclr/gc-largepages.yml

```diff
@@ -0,0 +1,52 @@
+# This file is bootstrapped from gc-longrunning.yml
```

Comment:
You would need to create a pipeline in azure devops that points to this yaml.

Reply:
I assume this will need to wait until this change is merged, and also we need to have certain permission in order to perform this task?

Reply:
Yeah, once it is merged. If you don't have permissions you can ping me or @trylek or someone on the infra crew for help creating it.

Reply:
Had an offline discussion with Bruce, here are some key points.

1. I think you should be able to create a pipeline and point it at your PR branch.
2. For the build test job, you should still be able to build the tests on OSX or Linux; they don't need to be run there
3. ​​The pipeline YML has a helixQueueGroup line. You'll need your own that will cause only Windows machines with your required setup to be used.
4. You'll probably need to add something to coreclr\templates\helix-queues-setup.yml that says which queues to use for your new helixQueueGroup name